media-gfx/ahoviewer/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
+    <email>juippis@gmail.com</email>
+  </maintainer>
```

Comment:
proxy-maint?

Reply:
Oh wow, thats a bit embarrasing from my behalf. The bot does see gentoo/proxy-maint so I guess that somehow dropped when I rebased and updated this PR from 1.6.0 to 1.6.1.